src/rest.js

```diff
@@ -95,8 +95,6 @@ function create(config, auth, className, restObject) {
 // REST API is supposed to return.
 // Usually, this is just updatedAt.
 function update(config, auth, className, objectId, restObject) {
-  enforceRoleSecurity('update', className, auth);
```

Comment:
Accident. Good catch. Somewhat worrying that it didn't cause any tests to fail though.